packages/types/src/Metadata/v2/Metadata.spec.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import extrinsicsFromMeta from '@polkadot/extrinsics/fromMetadata';
+
+import createType from '../../codec/createType';
+import Method from '../../primitive/Method';
+
+import Metadata from '../Metadata';
+import latestParsed from './latest.substrate.v2.json';
+import rpcData from './static';
+
+describe('MetadataV2', () => {
+  const metadata = new Metadata(rpcData);
+
+  it('decodes latest properly', () => {
+    const str = JSON.stringify(metadata.toJSON());
+
+    // console.error(str);
```

Comment:
These are really useful for manual inspection when having to add new stuff. (It is ok removing it for v0 and V1 since we won't update those anymore, but for the latest would prefer it in). The `str` value actually gets used for visual checks as well. (Yes, it _is_ noisy)